src/components/accession2/edit/CalculatorModal.tsx

```diff
@@ -10,7 +10,7 @@ import { Unit, WEIGHT_UNITS_V2 } from 'src/components/seeds/nursery/NewTest';
 import useSnackbar from 'src/utils/useSnackbar';
 import { Dropdown } from '@terraware/web-components';
 
-export interface CalculatorDialogProps {
+export interface CalculatorModalProps {
```

Comment:
i renamed some classes to match file names - also keeping naming consistent with some other modal files.